l2-frontend/src/fields/AnesthesiaProcess.vue

```diff
@@ -0,0 +1,350 @@
+<template>
+  <div>
+    <div class="sidebar-anesthesia"
+         :class="[{show_anesthesia: this.$store.state.showMenuAnesthesiaStatus}, {hide_anesthesia: !this.$store.state.showMenuAnesthesiaStatus}]">
+      <div class="title-anesthesia">
+        <div class="col-xs-10">Течение анестезии</div>
+        <div class="col-xs-2">
+          <button class="btn btn-blue-nb sidebar-btn" style="font-size: 12px" @click="show_anesthesia_sidebar">
+            <i class="glyphicon glyphicon-remove" v-tippy="{ placement : 'bottom'}" title="Закрыть"></i>
+          </button>
+        </div>
+      </div>
+      <div class="time-control">
+        <vue-timepicker style="margin-left: 3px" v-model="timeValue" format="H:mm" input-width="127px"
+                        input-class="timepicker_attr"
+                        hide-clear-button close-on-complete></vue-timepicker>
```

Comment:
Можно закрыть по простому тег, будет короче:
```suggestion
                        hide-clear-button close-on-complete />
```

Reply:
да